coral-service/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
include `:coral-service` in `ROOT_DIR/settings.gradle` file. You shouldnt need another set of gradlewrappers here.
If the reasoning behind this was a older gradle version, maybe update the gradle version for main project.

Reply:
This will be a part of [this discussion](https://github.com/linkedin/coral/pull/251#discussion_r852461057), including `coral-service` will add it as a submodule.